experimental/telemetry/tracker.go

```diff
@@ -105,3 +107,30 @@ func (t *tracker) TrackUserEvent(event, userID string, properties map[string]int
 	properties["UserActualID"] = userID
 	return t.TrackEvent(event, properties)
 }
+
+type telemetryAPIRequest struct {
+	Event      string
+	Properties map[string]interface{}
+}
+
+func (t *tracker) HandleClientEvent(r *http.Request) error {
```

Comment:
I'm not sure if that signature is better. With `(w http.ResponseWriter, r *http.Request)` is was easily plugable and with that change a consumer as to write there own error handling method. I would like to keep the methods a just plug an play.